FlareScoring.h missing

I'm trying to build the flysight viewer on an ubuntu box.
I get the error "flareform.cpp:11:26: fatal error: FlareScoring.h: No such file or directory", which makes sense since flareform.cpp has the line "#include "FlareScoring.h"", but there is no FlareScoring.h in the current source.

--- a/src/flareform.cpp
+++ b/src/flareform.cpp
@@ -8,7 +8,7 @@
 #include "datapoint.h"
 #include "mainwindow.h"
 #include "plotvalue.h"
-#include "FlareScoring.h"
+#include "flarescoring.h"
 
 FlareForm::FlareForm(QWidget *parent) :
     QWidget(parent),

--- a/src/flarescoring.cpp
+++ b/src/flarescoring.cpp
@@ -1,4 +1,4 @@
-#include "FlareScoring.h"
+#include "flarescoring.h"
 
 #include "mainwindow.h"
 

